eth/downloader/downloader.go

```diff
@@ -18,61 +18,64 @@
 package downloader
 
 import (
-	"bytes"
 	"errors"
 	"math"
 	"math/big"
-	"math/rand"
 	"sync"
 	"sync/atomic"
 	"time"
 
 	"github.com/ethereum/go-ethereum/common"
-	"github.com/ethereum/go-ethereum/core"
 	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/event"
 	"github.com/ethereum/go-ethereum/logger"
 	"github.com/ethereum/go-ethereum/logger/glog"
-	"gopkg.in/fatih/set.v0"
 )
 
 const (
-	eth60 = 60 // Constant to check for old protocol support
-	eth61 = 61 // Constant to check for new protocol support
+	eth61 = 61 // Constant to check for old protocol support
```

Comment:
Originally the protocol.go didn't have to care about these, as all the packets were different (opcodes). Since eth/62, both opcodes overlap between versions, as well as same opcode packets have differing contents, so I need to be able to differentiate at packet level already (until then I only needed it in the downloader). I think there are also a few references in the fetcher (1-2), but without the constants defined. It's a valid point you raised that it might be duplication, but I'd rather not leak out just 2-3 numbers used internally into the public API so maybe it can remain as is for now.

Regarding the wiki, it already defines these values (https://github.com/ethereum/wiki/wiki/Ethereum-Wire-Protocol#ethereum-sub-protocol) as the version of the protocol, which is negotiated during the sub-protocol setup, so I think it should be enough. But if I misunderstand you, do tell :)